app/styles/_print.scss

```diff
@@ -0,0 +1,19 @@
+.print-section { display: none; }
+
+@media print {
+
+  body {
+    background: white !important;
+    color: black !important;
+    font-size: 12pt;
+    margin: 0 5%;
+    &.ember-application { padding-left: 0; }
+  }
+
+  .print-section { display: block; text-align: center; }
+
+  .panel-footer, .btn, .nav, .view-current-title { display: none; }
```

Comment:
Each selector should be on their own line. In fact I thought we had a test for that in the Stylelint. Wonder why it's not failing the build? cc @billybonks